pom.xml

```diff
@@ -1739,7 +1739,7 @@
         <commons-logging.version>1.1.1</commons-logging.version>
         <commons-io.version>2.4.0.wso2v1</commons-io.version>
         <commons-httpclient.version>3.1</commons-httpclient.version>
-        <google.code.gson.version>2.8.6</google.code.gson.version>
+        <google.code.gson.version>2.8.5</google.code.gson.version>
```

Comment:
any reason for the downgrade?

Reply:
this was not used in the pack an it was at installed state